app/views/dashboards/show.html.erb

```diff
@@ -18,72 +18,107 @@
   </div>
 </div>
 <div class="container">
-  <div class="attending-pending my-4">
-    <div>
-      <h2>Pending tastings</h2>
+  <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" id="pills-home-tab" data-toggle="pill" href="#pills-home" role="tab"
+        aria-controls="pills-home" aria-selected="true">Attending</a>
+    </li>
+    <li class="nav-item" role="presentation">
+      <a class="nav-link" id="pills-contact-tab" data-toggle="pill" href="#pills-hostings" role="tab"
+        aria-controls="pills-hostings">Hosting</a>
+    </li>
+    <li class="nav-item" role="presentation">
+      <a class="nav-link" id="pills-contact-tab" data-toggle="pill" href="#pills-past" role="tab"
+        aria-controls="pills-past">Past</a>
+    </li>
+  </ul>
+
+  <div class="tab-content" id="pills-tabContent">
+  <div class="tab-pane fade show active" id="pills-home" role="tabpanel" aria-labelledby="pills-home-tab">
+    <div class="attending-pending">
+      <h3>Pending Tastings</h3>
       <div class="dashboard-cards ">
         <% @dashboard.pending_participations.each do |participation| %>
           <div class="dashboard-card">
-            <% if participation.tasting.photo.attached? %>
-              <%= cl_image_tag participation.tasting.photo.key%>
-            <% else %>
-              <%= image_tag participation.tasting.image%>
-            <% end %>
-            <h3><%= link_to participation.tasting.title, tasting_path(participation.tasting) %></h3>
-            <p><%= participation.tasting.date.to_time.strftime('%B %e at %l:%M %p') %></p>
+                  <% if participation.tasting.photo.attached? %>
+                    <%= cl_image_tag participation.tasting.photo.key%>
+                  <% else %>
+                    <%= image_tag participation.tasting.image%>
+                  <% end %>
+                <h3>
+                <%= link_to participation.tasting.title, tasting_path(participation.tasting) %>
```

Comment:
If you really want the content the be on a different line than the tag, you should indent it. You can keep it all inline if it is short.
You can review the other lines - some are indented, some not.